### DISCUSSION TITLE
使用arco-design-vue创建的项目，页面几分钟没去访问，从新再去访问就会报错，重新刷新页面又正常了

### DISCUSSION THREAD

**Discussion by asher886** [arco-design/arco-design-vue#1171] [Q&A]
![111111111](https://user-images.githubusercontent.com/99728849/169279963-a6bc424c-89c4-47b1-a3f7-b6cea106ab5a.png)

使用arco-design-vue创建的项目，页面几分钟没去访问，从新再去访问就会报错，重新刷新页面又正常了

**Comment by flsion** [comment]
这个没有遇到过，看着像是 router 中加载文件失败了
